tests/sentry/integrations/gitlab/test_integration.py

```diff
@@ -26,13 +26,20 @@ class GitlabIntegrationTest(IntegrationTestCase):
     }
     default_group_id = 4
 
+    def setUp(self):
+        super(GitlabIntegrationTest, self).setUp()
+        self.init_path_without_guide = '%s%s' % (self.init_path, '?completed_installation_guide')
```

Comment:
Does `'completed_installation_guide'` need to have a value i.e. `'completed_installation_guide=true`?